src/main/resources/site/parts/statisticHeader/statisticHeader.ts

```diff
@@ -0,0 +1,97 @@
+import { type Content, get as getContentByKey } from '/lib/xp/content'
+import { getContent, pageUrl } from '/lib/xp/portal'
+import { sleep } from '/lib/xp/task'
+import { render as r4xpRender } from '/lib/enonic/react4xp'
+import { type Phrases } from '/lib/types/language'
+import { randomUnsafeString } from '/lib/ssb/utils/utils'
+
+import { getStatisticsDates, getStatisticTitle } from '/lib/ssb/utils/statisticsUtils'
+import { StatisticInListing } from '/lib/ssb/dashboard/statreg/types'
+
+import { getStatisticByIdFromRepo } from '/lib/ssb/statreg/statistics'
+import { getPhrases } from '/lib/ssb/utils/language'
+import { renderError } from '/lib/ssb/error/error'
+import { hasWritePermissionsAndPreview } from '/lib/ssb/parts/permissions'
+import { currentlyWaitingForPublish as currentlyWaitingForPublishOld } from '/lib/ssb/dataset/publishOld'
+import { type StatisticHeader } from '/lib/types/partTypes/statisticHeader'
+import { type Statistics, type OmStatistikken } from '/site/content-types'
+
+export function get(req: XP.Request): XP.Response {
+  try {
+    return renderPart(req)
+  } catch (e) {
+    return renderError(req, 'Error in part: ', e)
+  }
+}
+
+export function preview(req: XP.Request): XP.Response {
+  return renderPart(req)
+}
+
+// eslint-disable-next-line complexity
+function renderPart(req: XP.Request): XP.Response {
+  const page = getContent<Content<Statistics>>()
+  if (!page) throw Error('No page found')
+
+  const phrases = getPhrases(page) as Phrases
+  const wait: number =
+    app.config && app.config['ssb.statistics.publishWait'] ? parseInt(app.config['ssb.statistics.publishWait']) : 100
```

Comment:
Is it possible to use `app?.config?.['ssb.statistics.publishWait']? Same with `ssb.statistics.publishMaxWait`.